docs/guides/platform/Install_MERLIN_In_AirGap.md

```diff
@@ -209,12 +200,6 @@ mkdir -p /etc/docker/certs.d/<the_registry_host_name>:<the_registry_host_port>
 cp /opt/registry/certs/domain.crt /etc/docker/certs.d/<the_registry_host_name>:<the_registry_host_port>/ca.crt
 ```
 
-Log in the Docker registry.
-
-```bash
-docker login <the_registry_host_name>:<the_registry_host_port> -u <registry_user_name> -p <registry_password>
-```
-
 #### Configure the registry
 
 After creates the registry, configure the Docker registry:
```

Comment:
After creat*ing* ...

Reply:
@edmundreinhardt Good catch. It's resolved. Thank you.